pkgs/by-name/co/cosmic-edit/package.nix

```diff
@@ -21,12 +21,11 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cosmic-edit";
-  version = "unstable-2024-03-30";
-
```

Comment:
What happened to this line?  We usually leave a blank one between `version` and `src`, like there was before.

Reply:
Did it need to be blank? I see sometimes that it is there and sometimes no? It is in the same {} so I don't think it should be.

Reply:
Added back.